airflow/models.py

```diff
@@ -201,10 +202,11 @@ def get_dag(self, dag_id):
                     dag.last_loaded < orm_dag.last_expired
                 )
         ):
-            # Reprocessing source file
+            # Then reprocess source file
```

Comment:
I changed it to match https://github.com/apache/incubator-airflow/pull/1678/files#diff-a32a363fa616685db3bfefba947535b2L195 ("If the root_dag_id is absent or expired...then reprocess source file")